inc/classes/busting/trait-file-busting.php

```diff
@@ -85,7 +85,7 @@ public function delete() {
 	 * @return string|bool Version of the file. False if the file does not exist.
 	 */
 	protected function get_busting_version() {
-		if ( isset( $this->file_version ) ) {
+		if ( isset( $this->file_version ) && empty( $this->file_version ) ) {
```

Comment:
`empty()` is enough here as it's a combo of `isset` and not falsey.

Are we want to check that it's empty or that it's not empty?

Reply:
You are totally right, I wanted to check if this file_version is not empty.
Solved now